docs/docs/notification-center/react-components.md

```diff
@@ -171,51 +171,52 @@ The `i18n` prop can accept 2 different types of values
       <div>
         <ul>
           <li><code>ar</code> (Arabic)</li>
+          <li><code>as</code> (Assamese)</li>
+          <li><code>bg</code> (Bulgarian)</li>
+          <li><code>bn</code> (Bengali)</li>
+          <li><code>ca</code> (Catalan)</li>
+          <li><code>cs</code> (Czech)</li>
+          <li><code>da</code> (Danish)</li>
           <li><code>de</code> (German)</li>
+          <li><code>el</code> (Greek)</li>
           <li><code>en</code> (English)</li>
+          <li><code>es</code> (Spanish)</li>
           <li><code>fa</code> (Farsi)</li>
           <li><code>fi</code> (Finnish)</li>
           <li><code>fr</code> (French)</li>
           <li><code>gu</code> (Gujarati)</li>
+          <li><code>he</code> (Hebrew)</li>
           <li><code>hi</code> (Hindi)</li>
-					<li><code>id</code> (Indonesian)</li>
+          <li><code>hr</code> (Croatian)</li>
+          <li><code>hu</code> (Hungarian)</li>
+          <li><code>id</code> (Indonesian)</li>
+          <li><code>ig</code> (Igbo)</li>
           <li><code>it</code> (Italian)</li>
+          <li><code>ja</code> (Japanese)</li>
           <li><code>ka</code> (Kannada)</li>
+          <li><code>ko</code> (Korean)</li>
+          <li><code>ku</code> (Kurdish)</li>
+          <li><code>ml</code> (Malayalam)</li>
           <li><code>mr</code> (Marathi)</li>
           <li><code>ne</code> (Nepali)</li>
-          <li><code>ru</code> (Russian)</li>
-          <li><code>es</code> (Spanish)</li>
-          <li><code>uk</code> (Ukrainian)</li>
```

Comment:
Did you change the order to alphabetical here? Not sure why this diff was generated 😃

Reply:
Oh nvm, just seen the commit message 🎉